utils/general.ts

```diff
@@ -44,3 +44,13 @@ export const toCopyContent = (
 
   return procesedInnerText;
 };
+
+export const findFirstLvlNav = (locPath: string): string => {
```

Comment:
Can you add a doc block for this fn plz.

Also can you add some inline comments explaining why you're special casing `"ver"`. It doesn't appear that we have tests for these util functions but it would be nice, even just so that "future dev" knows what the expected functionality of the fn is.